launcher/ui/WinDarkmode.h

```diff
@@ -0,0 +1,70 @@
+#ifndef WINDARKMODE_H
+#define WINDARKMODE_H
+
+#include <QWidget>
```

Comment:
is this needed in here? if not, it's usually better to put includes only in cpp files rather than header ones :)